<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonEditDialog</class>
 <widget class="QDialog" name="PersonEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Фамилия</string>
       </property>
       <property name="buddy">
        <cstring>leFamilyName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFamilyName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="buddy">
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Отчество</string>
       </property>
       <property name="buddy">
        <cstring>leSurName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSurName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Дата рождения</string>
       </property>
       <property name="buddy">
        <cstring>deBirthDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="deBirthDate">
       <property name="specialValueText">
        <string>Дата рождения</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Паспорт</string>
       </property>
       <property name="buddy">
        <cstring>lePassport</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassport">
       <property name="inputMask">
        <string>99 99 999999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Водительское удостверение</string>
       </property>
       <property name="buddy">
        <cstring>leLicense</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLicense">
       <property name="inputMask">
        <string>99 &gt;NN 999999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
